examples/manual-creation.rs

```diff
@@ -30,12 +30,12 @@ fn main() {
     let mut events_loop = glutin::EventsLoop::new();
     let window = glutin::WindowBuilder::new();
     let context = glutin::ContextBuilder::new();
-    let gl_window = Rc::new(glutin::GlWindow::new(window, context, &events_loop).unwrap());
+    let gl_window = Rc::new(glutin::WindowedContext::new_windowed(window, context, &events_loop).unwrap());
```

Comment:
Nit: go `glutin::ContextBuilder::new().new_windowed(...).unwrap()`, instead of `glutin::Context::new_windowed(..).unwrap()`.